.github/workflows/ci.yml

```diff
@@ -1,10 +1,16 @@
 name: CI
-on: [push, pull_request]
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
 jobs:
-  build:
+  test:
     strategy:
       matrix:
-        ruby: ['2.5', '2.6', '2.7', '3.0', '3.1', head]
```

Comment:
Wouldn't it be better to keep 'head' around? That way when Ruby 3.3 (and future) version is out, there is chance that jekyll-toc will be ready.

Reply:
`head` might be unstable, so I didn't add ruby-head to CI, but your point makes sense, so I've added ruby-head to CI in f24235ee6350d49768662d2c43a09e2fca28e1ec. :+1: